src/app/components/clients/clients.component.ts

```diff
@@ -14,8 +14,10 @@ import { DeleteDialogComponent } from '../dialogs/delete-dialog/delete-dialog.co
 })
 
 export class ClientsComponent implements OnInit {
-  allClients: IClient[];
+  allClients: Client[];
   errorMessage: string;
+   Client = <Client>{};
```

Comment:
client: Client  - e sintaxa

Reply:
pentru ce e folosit asta?

am mai vazut si in alte componente declarat ceva de genu.

Reply:
Empty object de tip Client, pentru this.Client.clientName = data;

Reply:
si unde folosesti clientName? in componenta nu am gasit nimic, nici in view.